actions/post_actions.jsx

```diff
@@ -126,16 +125,28 @@ export async function createPost(post, files, success) {
     }
 
     if (post.root_id) {
-        PostStore.storeCommentDraft(post.root_id, null);
+        dispatch(storeCommentDraft(post.root_id, null));
     } else {
-        PostStore.storeDraft(post.channel_id, null);
+        dispatch(storeDraft(post.channel_id, null));
```

Comment:
Should delete that functions from stores/post_stores

Reply:
We have a ticket to just delete the whole store soon